### PR TITLE
Automatically bump secrethub.ClientVersion on release branch

### PR DESCRIPTION
**Pull request by florisvdg**
image

### CODE REVIEW

File: .github/workflows/bump-version.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+jobs:
+  bump-version:
+    name: Bump secrethub.ClientVersion
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Bump version
+        uses: florisvdg/action-version-bump@master
```
Comment (SimonBarendse) [inline .github/workflows/bump-version.yml:14]: Shall we use a pinned version?

Comment (florisvdg) [reply .github/workflows/bump-version.yml:14]: I didn't create a release for it yet, but when I do I'll pin it here. But seeing as it's not a critical CI/CD component (you can always set it manually) I'll leave it like this for now.

Comment (SimonBarendse) [reply .github/workflows/bump-version.yml:14]: Our branch protection policy doesn't allow a branch to be merged when any of the checks fails. Can we set it optional somehow?

Comment (SimonBarendse) [reply .github/workflows/bump-version.yml:14]: Is there a reason not to release a version yet?

Comment (florisvdg) [reply .github/workflows/bump-version.yml:14]: It's excluded: 
image

Comment (SimonBarendse) [reply .github/workflows/bump-version.yml:14]: Yes, that means if no status is reported at all, that's fine.
However, if the action reports a failure status, the PR isn't allowed to be merged.

Comment (florisvdg) [reply .github/workflows/bump-version.yml]: Okay pinned it.

**Pull request merged**